packages/neos-ui/src/Containers/Modals/AddNode/index.js

```diff
@@ -53,6 +54,16 @@ const calculateActiveMode = (currentMode, allowedNodeTypesByMode) => {
 }))
 export default class AddNodeModal extends PureComponent {
     static propTypes = {
+        reference: PropTypes.shape({
+            subject: PropTypes.shape({
+                contextPath: PropTypes.string.isRequired,
+                fusionPath: PropTypes.string
+            }).isRequired,
+            collection: PropTypes.shape({
```

Comment:
I'd prefer `closestContentCollection`, but I don't like bloating this, since it can't be anything else in this context. I'd rather change the name of `reference` to make more clear what it is about. Maybe something like `creationContext`.